docs/specification/draft/basic/authorization.mdx

```diff
@@ -373,3 +460,15 @@ MCP clients **MUST** implement and use the `resource` parameter as defined in [R
 to explicitly specify the target resource for which the token is being requested. This requirement aligns with the recommendation in
 [RFC 9728 Section 7.4](https://datatracker.ietf.org/doc/html/rfc9728#section-7.4). This ensures that access tokens are bound to their intended resources and
 cannot be misused across different services.
+
+### Scope Minimization
+
+Following the principle of least privilege, MCP clients **SHOULD** request only the minimum scopes necessary for their intended operations.
+The `scopes_default` extension in Protected Resource Metadata provides server guidance for appropriate default scope selection, helping clients avoid:
```

Comment:
```suggestion
The `scopes_default` field in the authorization server's Protected Resource Metadata document provides server guidance for appropriate default scope selection, helping clients avoid:
```

Reply:
@nbarbettini - but the AS is not the one that provides the PRM, it's the MCP server?